src/chi/components/input-wrapper.scss

```diff
@@ -48,4 +62,19 @@ $border: 0.0625rem;
       }
     }
   }
+
+  & .m-label__wrapper {
+    align-items: center;
+    display: flex;
+    justify-content: start;
+    margin-bottom: 0.5rem;
```

Comment:
Can we change this value to `margin-bottom: 0.25rem;` on both `.m-label__wrapper` and .`chi .a-label, .chi label`